cfme/tests/physical_infrastructure/test_redfish_physical_chassis.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.redfish import RedfishProvider
+
+pytestmark = [
+    pytest.mark.provider([RedfishProvider], scope="function"),
+]
+
+
+@pytest.fixture(scope="function")
+def physical_chassis(appliance, provider, setup_provider_funcscope):
```

Comment:
Out of curiosity: what does `setup_provider_funcscope` do? Sounds like we're forcibly creating provider everytime, not sure if that's intended?

Reply:
As [required](https://github.com/ManageIQ/integration_tests/pull/7726#discussion_r224406426), these fixtures are restricted to funcion scope, thus `funscope` suffix instead of `modscope`.